spec/controllers/projects/queries_controller_spec.rb

```diff
@@ -328,7 +328,7 @@
         it "passes variables to template" do
           allow(controller).to receive(:render).and_call_original
 
-          post "toggle_public", params: { id: 42 }
+          post "toggle_public", params: { id: 42, value: 1 }
```

Comment:
No spec with `value: 0`, only one without any to check authentication requirement

Reply:
done